torchvision/prototype/features/_feature.py

```diff
@@ -83,3 +90,159 @@ def __torch_function__(
             return cls.new_like(args[0], output, dtype=output.dtype, device=output.device)
         else:
             return output
+
+    def horizontal_flip(self) -> Any:
+        # Just output itself
+        # How dangerous to do this instead of raising an error ?
+        return self
+
+    def vertical_flip(self) -> Any:
+        # Just output itself
+        # How dangerous to do this instead of raising an error ?
+        return self
+
+    def resize(  # type: ignore[override]
```

Comment:
Why `ignore[override]` is needed here?

Reply:
there is torch.Tensor built-in deprecated op: Tensor.resize : https://github.com/pytorch/pytorch/blob/e8727994eb7cdb2ab642749d6549bc497563aa06/torch/_tensor.py#L588-L593

Reply:
holly molly! that's not good. Does it make sense to rename the method? I can see that Tensor.resize_ exists still on the [docs](https://pytorch.org/docs/stable/generated/torch.Tensor.resize_.html?highlight=tensor%20resize#torch.Tensor.resize_) but can't find the resize.

Reply:
non-inplace Tensor.resize is deprecated -> no docs. IMO, we can keep both.

Reply:
Add a big TODO to get feedback on this. cc @NicolasHug thoughts?

Reply:
@NicolasHug I agree that the TODO note is unclear. I'll update in a follow-up PR. The point is to fix `# type: ignore[override]` and make mypy happy.

Reply:
This is a good question. Maybe it is impossible. I was thinking about mypy overload: https://mypy.readthedocs.io/en/stable/more_types.html#function-overloading if this could help...